README.md

```diff
@@ -70,41 +75,69 @@ It is any form of learning that uses the internet as a means of delivering lesso
+**3. Is the innovation simpler to use, or more convenient or more affordable than the incumbent’s existing offering?**
 
-A: **YES**
+Yes. Its mere presence on the internet fulfills all the above conditions, excepting simplicity. Internet usage requires a measure of skill on its own that certain adult demographics do not have. However, in general, its accessibility makes up for it. MOOCs are accessible from a user’s own home, which also touches on affordability. Even if the prices were the same for an MOOC and a physical class, if we view price outside of the simplistic context of money and add in other resources expended to attend physical college classes, such as travel time, energy expenditure, and psychological stressors, MOOCs win out on affordability.
 
-4)	Can we create a business model that enables us to earn attractive profits at the discount prices required to win the business of these overserved customers at the low end?
+However, quality of education for cost may suffer: online courses are yet to be as widely accepted as physical classing.
 
-A: **YES**
+**4. Does the offering have a technological enabler that can carry it to upmarket and allow it to improve?**
 
--	Set 3:
-2)	Is the innovation disruptive to all of the significant incumbent firms in the industry? If it appears to be sustaining to one or more significant players in the industry, then the odds will be stacked in that firm’s favor, and the entrant is unlikely to win.
+Yes: the internet, and the communication technologies that carry it.
+  
+**5. Is the technology paired with a business model innovation that allows it to be sustainable?**
+
+Yes. Online marketing and digital commerce have been around for decades, starting with AuctionWeb in 1995. Online transactions have dramatically improved since then, allowing people to purchase practically anything online, if they know where to look.
+
+**6. Are existing providers motivated to ignore the new innovation and not threatened at the outset?**
+
+Technically, no. However, due to the social stigma of online classes, they aren’t immediately threatened by it: socially speaking, someone graduating from an MOOC would not have the same level of social awe as a graduate from a top-end university such as Harvard University. However, the innovation is not being ignored by existing providers: MOOCs often include courses taught by these same high-end universities. Existing providers, such as UP, are also adopting the new innovation into their systems.
 
-A: **YES**
 
 (Christensen, 1997)
 
+## HOW DOES ONLINE LEARNING DISRUPT TRADITIONAL EDUCATION? ## 
+
+According to statistics taken from Purdue University & The National Center for Education Statistics, more than 6 million students are participating in online courses as part of their higher education program in 2018 and the numbers are rapidly increasing each year (Online Learning - What is it and how does it work?, 2017).
+
 ### Sources: ###
```

Comment:
- Adams, S. (Apr. 26, 2019). Online Education Provider Coursera Is Now Worth More Than $1 Billion. Retrieved from https://www.forbes.com/sites/susanadams/2019/04/25/online-education-provider-coursera-is-now-worth-more-than-1-billion/?sh=45c2902130e1

- Christensen Institute. (Oct. 24, 2017). Are you (really) disruptive? Six ways to find out. Retrieved from https://www.christenseninstitute.org/blog/really-disruptive-six-ways-find/

- Harvard Business Review (Mar. 30, 2020), Disruptive Innovation Explained, retrieved: November 18, 2020, from: https://www.youtube.com/watch?v=qDrMAzCHFUU

- Interaction Design Foundation, 2nd Edition (Jan. 01, 2014), The Encyclopedia of Human-Computer Interaction, retrieved: November 21, 2020, from: https://www.interaction-design.org/literature/book/the-encyclopedia-of-human-computer-interaction-2nd-ed/disruptive-innovation

- Iskandar, Andrea. (2016). Coursera's Business Level Strategy. 10.13140/RG.2.1.2902.8088. 

- Malamed, C(n.d.), Glossary of Online Learning Terms, retrieved: November 21, 2020, from: https://theelearningcoach.com/resources/online-learning-glossary-of-terms/

- Online Learning - What is it and how does it work? (2017). Educations.com. retrieved: November 27, 2020, from: https://www.educations.com/study-guides/study-online/

- QUT IFB101(Mar. 22, 2015), Disruptive Innovation, retrieved: November 16, 2020, from: https://www.youtube.com/watch?v=Cu6J6taqOSg

- Stern, J. (n.d.). Introduction to Online Teaching and Learning, retrieved: November 26, 2020, from: http://www.wlac.edu/online/documents/otl.pdf

- University), V. U.-E. (Global B. S. &. (n.d.). E-learning as a Disruptive Innovation. Vision U-EENI (Global Business School & University), retrieved: November 28, 2020, from: http://www.u-eeni.edu.es/en/e-learning-Disruptive-Innovation.shtml